parser-typechecker/src/Unison/Codebase/Branch.hs

```diff
@@ -391,20 +393,20 @@ toCausalRaw = \case
   Branch (Causal.Merge _h e tls)     -> RawMerge (toRaw e) (Map.keysSet tls)
 
 -- returns `Nothing` if no Branch at `path` or if Branch is empty at `path`
-getAt :: Path
+getAt :: Path pos
       -> Branch m
       -> Maybe (Branch m)
-getAt path root = case Path.uncons path of
+getAt path root = case Path.uncons (Path.unsafeToRelative path) of
```

Comment:
We know that this path is relative to this branch since they were passed together. If anyone has a better idea than casting like this in trusted spots, let me know 😄